const registerUser = async (req, res, next) => {
  try {
  } catch (error) {}
};

const loginUser = async (req, res, next) => {
  try {
  } catch (error) {}
};

export { registerUser, loginUser };
